src/nvim/quickfix.c

```diff
@@ -4026,8 +4080,20 @@ static void qf_fill_buffer(qf_list_T *qfl, buf_T *buf, qfline_T *old_last)
       }
       lnum = buf->b_ml.ml_line_count;
     }
+
+    qftf_list = call_qftf_func(qfl, qf_winid, (long)(lnum + 1),
+                               (long)qfl->qf_count);
```

Comment:
`qf_count` is int.
`lnum` uses `linenr_T` type which is `long`.

Reply:
redefine `call_qftf_func` and use `int` instead of `long`  or just change `(long)(lnum + 1)` to `lnum + 1` directly?

Reply:
Use it directly.